.github/workflows/ci.yaml

```diff
@@ -278,7 +292,8 @@ jobs:
           name: golem-binaries
           retention-days: 1
           path: |
-            target/release/golem-service
+            target/release/golem-template-service
+            target/release/golem-worker-service
             target/release/golem-service-yaml
```

Comment:
there should be:
 
target/release/golem-template-service-yaml
target/release/golem-worker-service-yaml

Reply:
No need of this now. There is an automated merge now

Reply:
Merge of yanls

Reply:
ok